templates/redis/redis-statefulset.yaml

```diff
@@ -31,6 +31,10 @@ spec:
 {{- with .Values.labels }}
 {{ toYaml . | indent 8 }}
 {{- end }}
+      annotations:
+        {{- if .Values.redis.safeToEvict }}
```

Comment:
As we upgrade charts, this will be `null`, so evaluate to `false` and not show, unless we explicitly override from Houston's `ConfigMap`.